### ISSUE TITLE
Errors while running main.py

### ISSUE THREAD

**Issue by naijaexboy** [Princekrampah/learningFastAPI#1]
I was learning from your youtube channel and there seems to be an error when I compile and run the main.py.

emails.py", line 12, in <module>
    conf = ConnectionConfig(
  File "pydantic\env_settings.py", line 39, in pydantic.env_settings.BaseSettings.__init__
  File "pydantic\main.py", line 342, in pydantic.main.BaseModel.__init__pydantic.error_wrappers.ValidationError: 5 validation errors for ConnectionConfig
MAIL_STARTTLS
  field required (type=value_error.missing)
MAIL_SSL_TLS
  field required (type=value_error.missing)
MAIL_FROM
  value is not a valid email address (type=value_error.email)
MAIL_SSL
  extra fields not permitted (type=value_error.extra)
MAIL_TLS
  extra fields not permitted (type=value_error.extra)

What can I do to fix this error while running from vscode?

**Comment by Princekrampah** [comment]
Hello,

I think you didn't provide all values as needed as fields are required, but
you haven't provided them. This is according to the error output.

Try making sure you have provided all values as required.

Best regards,
Prince

**Comment by naijaexboy** [comment]
Hello,

Thank you for the feedback.

I actually did that, I am not sure whether there is a need for any additional configuration as all I did was to run your script exactly the way you wrote it but I got the error. 

All the values are there the way you placed it in your email.py and main.py modules.

Is there anything specific I should check or any environmental settings?

**Comment by Princekrampah** [comment]
Yes, you have to add a .env file with holds your environment variables.

I didn't upload that to git, it is confidential. Make sure you have that
setup.
